### ISSUE TITLE
An error has occured : java.lang.IllegalArgumentException: source is null

### ISSUE THREAD

**Issue by riverdusty** [plantuml/plantuml#426]
Installed Tomcat version 9 x64 on Windows x64.
Put plantuml.war file into webapps Tomcat directory. (Plantuml 1.2020.23 apparently).
Take example diagram from plantuml website to test.
```
@startuml
class Dummy {
  String data
  void methods()
}

class Flight {
   flightNumber : Integer
   departureTime : Date
}
@enduml
```

POST to 'http://localhost:8090/plantuml/svg' (I changed the port but I doubt that has anything to do with the error).
(Using VisualAge Smalltalk to do the post, but test with other systems (Pharo specifically)).

Response is 

```
An error has occured : java.lang.IllegalArgumentException: source is null
In an infinite Universe anything can happen.
Diagram size: 11 lines / 144 characters.
PlantUML (1.2020.23) cannot parse result from dot/GraphViz.
Please go to https://plantuml.com/graphviz-dot to check your GraphViz version.
Java Runtime: Java(TM) SE Runtime Environment
JVM: Java HotSpot(TM) 64-Bit Server VM
Default Encoding: Cp1252
Language: en
Country: ZA
PLANTUML_LIMIT_SIZE: 4096
This may be caused by :
- a bug in PlantUML
- a problem in GraphViz
You should send this diagram and this image to
plantuml@gmail.com
or
post to
https://plantuml.com/qa
to solve this issue.
You can try to turn arround this issue by simplifing your diagram.
java.lang.IllegalArgumentException: source is null
net.sourceforge.plantuml.brotli.BrotliInputStream.<init>(BrotliInputStream.java:73)
net.sourceforge.plantuml.brotli.BrotliInputStream.<init>(BrotliInputStream.java:53)
net.sourceforge.plantuml.windowsdot.WindowsDotArchive.extract(WindowsDotArchive.java:72)
net.sourceforge.plantuml.windowsdot.WindowsDotArchive.getWindowsExeLite(WindowsDotArchive.java:91)
net.sourceforge.plantuml.cucadiagram.dot.GraphvizWindows.specificDotExe(GraphvizWindows.java:58)
net.sourceforge.plantuml.cucadiagram.dot.GraphvizWindows.searchDotExe(GraphvizWindows.java:50)
net.sourceforge.plantuml.cucadiagram.dot.AbstractGraphviz.<init>(AbstractGraphviz.java:76)
net.sourceforge.plantuml.cucadiagram.dot.GraphvizWindows.<init>(GraphvizWindows.java:72)
net.sourceforge.plantuml.cucadiagram.dot.GraphvizUtils.create(GraphvizUtils.java:80)
net.sourceforge.plantuml.svek.DotStringFactory.getGraphvizVersionInternal(DotStringFactory.java:287)
net.sourceforge.plantuml.svek.DotStringFactory.getGraphvizVersion(DotStringFactory.java:281)
net.sourceforge.plantuml.svek.GeneralImageBuilder.printEntityInternal(GeneralImageBuilder.java:564)
net.sourceforge.plantuml.svek.GeneralImageBuilder.printEntity(GeneralImageBuilder.java:546)
net.sourceforge.plantuml.svek.GeneralImageBuilder.printEntities(GeneralImageBuilder.java:538)
net.sourceforge.plantuml.svek.GeneralImageBuilder.buildImage(GeneralImageBuilder.java:411)
net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFileInternal(CucaDiagramFileMakerSvek.java:114)
net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFile(CucaDiagramFileMakerSvek.java:79)
net.sourceforge.plantuml.cucadiagram.CucaDiagram.exportDiagramInternal(CucaDiagram.java:655)
net.sourceforge.plantuml.classdiagram.ClassDiagram.exportDiagramInternal(ClassDiagram.java:201)
net.sourceforge.plantuml.UmlDiagram.exportDiagramNow(UmlDiagram.java:176)
net.sourceforge.plantuml.AbstractPSystem.exportDiagram(AbstractPSystem.java:140)
net.sourceforge.plantuml.servlet.DiagramResponse.sendDiagram(DiagramResponse.java:112)
net.sourceforge.plantuml.servlet.UmlDiagramService.doDiagramResponse(UmlDiagramService.java:101)
net.sourceforge.plantuml.servlet.UmlDiagramService.doPost(UmlDiagramService.java:88)
javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:888)
org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1597)
org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
java.lang.Thread.run(Unknown Source)
```

**Comment by arnaudroques** [comment]
Many thanks for the report.
This should be fixed in last release 1.2020.26
Tell us if it's not working for you !

**Comment by riverdusty** [comment]
Confirmed. 1.2020.26 fixes this issue.
Many thanks for the quick response
